api/agent/agent.go

```diff
@@ -231,6 +238,11 @@ func (a *agent) Submit(callI Call) error {
 	ctx = opentracing.ContextWithSpan(context.Background(), span)
 	call.End(ctx, err)
 
+	err = a.fireAfterCall(ctx, call.Model())
+	if err != nil {
+		return fmt.Errorf("AfterCall: %v", err)
```

Comment:
this could throw things off a little, I guess, since an error from `exec` may get masked by this function. say, timeout or docker auth failed. I guess we want to leave the option for users to be able to return their own errors to clients, though, so this makes sense too. we can still do https://github.com/fnproject/fn/pull/424/files#r144922405 and users can access underlying error.